src/Traits/Subsection.php

```diff
@@ -90,7 +90,7 @@ protected static function get_link( $link ) {
 	 */
 	protected static function get_link_text( $link_text ) {
 		$link = $link_text['link'] ?? array();
-		$text = $link_text['text'][ self::get_locale() ] ?? $link_text['text']['en'] ?? '';
+		$text = $link_text[ self::get_locale() ] ?? $link_text['en'] ?? '';
```

Comment:
This should still have the ['text'] segment. Compare to the[ json file for Avarda for example](https://github.com/krokedil/krokedil-settings-page-configs/blob/main/configs/avarda-checkout-for-woocommerce.json#L37-L51)